Libraries/LibTLS/Record.cpp

```diff
@@ -77,12 +77,12 @@ void TLSv12::update_packet(ByteBuffer& packet)
 
             if (m_context.crypto.created == 1) {
                 // `buffer' will continue to be encrypted
-                auto buffer = ByteBuffer::create_zeroed(length);
+                auto buffer = ByteBuffer::create_uninitialized(length);
```

Comment:
Hmm?

Reply:
There's no need for that to be zeroed, it's initialised right there

`buffer`:
- L91: 0:(`packet.size() - header_size`)
- L98: (`packet.size() - header_size`):(`packet.size() - header_size + mac_size`)
- L102: (`packet.size() - header_size + mac_size`):`buffer.size()`
- (asserted at L103)

`ct`:
- L88: 0:(`header_size - 2`)
- L123: (`header_size - 2`):(`header_size`)
- L111: (`header_size`):(`header_size + iv_size`)
- L117: (`header_size + iv_size`):(`header_size + iv_size + length`)
- (asserted at L113)

Reply:
I believe you, just seems like a sneaky unrelated change not mentioned in commit message or anywhere :)

Reply:
I'll split it up

(that also served to prove it to myself...)